README.md

```diff
@@ -2,7 +2,7 @@
 
 A .NET Port of the [Scientist](https://github.com/github/scientist) library for carefully refactoring critical paths. 
 
-[![Build status](https://ci.appveyor.com/api/projects/status/b548cd5okkel3h4x/branch/master?svg=true)](https://ci.appveyor.com/project/shiftkey/scientist-net/branch/master)
+[![3.0.0 Unit Test](https://github.com/marblekirby/Scientist.net/actions/workflows/ver3-unit-test.yml/badge.svg?branch=3.0.0)](https://github.com/marblekirby/Scientist.net/actions/workflows/ver3-unit-test.yml)
```

Comment:
We should have something in place to make sure we update this once it gets merged

Reply:
I'd suggest using what it should be when it gets merged in rather than updating it afterwards perhaps?

Reply:
Sure do I need to make any changes?

Reply:
I think this would work:
```suggestion
[![Build status](https://ci.appveyor.com/api/projects/status/b548cd5okkel3h4x/branch/3.0.0?svg=true)](https://ci.appveyor.com/project/shiftkey/scientist-net/branch/3.0.0)
```